content/concepts/asset-manager.md

```diff
@@ -1934,11 +1963,11 @@ Below is the JSON format representing an example of an asset of the Asset Manage
         "data": ["red","purple"]
       }
     ],
-    "end_of_use_date": [
+    "model_height": [
       {
         "locale": null,
         "channel": "ecommerce",
-        "data": "02/03/2021"
+        "data": "1.65m"
```

Comment:
why not keeping a date attribute example?

Reply:
The previous example illustrated a localizable text value by using a date. We preferred changing the example to avoid confusion with the new date attribute type.